contrib/!config/dockerfile/README.org

```diff
@@ -0,0 +1,26 @@
+#+TITLE: Dockerfile contribution layer for Spacemacs
+
+[[file:img/docker.png]]
+
+* Table of Contents                                                   :TOC@4:
+ - [[#description][Description]]
```

Comment:
Well yeah, actually that's sth to take into account even if we didn't use the `toc-org` package. I guess it could be fixed in the https://github.com/github/markup repo